configs/vitest.config.ts

```diff
@@ -45,6 +45,12 @@ export default defineConfig(({ mode }) => {
     },
     resolve: {
       conditions: testSSR ? ["node"] : ["browser", "development"],
+      // fix issues with stores and node@20:
+      ...(!testSSR && { 
+        alias: {
+          "solid-js/store": "solid-js/store/dist/store.js"
```

Comment:
Could you explain this? do we need the same thing for `"solid-js/web"` too? Also we have set the required node version to `18` for the repo. Maybe we should add a [`.nvmrc`](https://github.com/thetarnav/solid-devtools/blob/main/.nvmrc) file to enforce it?

Reply:
Node@20 partially messes up the resolver for vitest. I have not yet found the reasons, only two workarounds: make solid an external dependency or alias some files.